src/contracts/AludelFactory.sol

```diff
@@ -117,6 +117,7 @@ contract AludelFactory is Ownable, InstanceRegistry {
 
         // transfer ownership
         Ownable(aludel).transferOwnership(ownerAddress);
+        emit ProgramAdded(address(aludel), name, stakingTokenUrl);
```

Comment:
We actually rely on AludelCreated from [Aludel.sol](https://github.com/alchemistcoin/alchemist/blob/main/contracts/aludel/Aludel.sol#L298), you think it's worth having two events on the same transaction for seemly the same thing?

Reply:
The deployment scripts relied on the InstanceAdded event from the InstanceRegistry, does the frontend rely on AludelCreated?

I think it's more fitting to emit the event in the factory, since its main responsibility is to permission the creation of Aludels, and it'd enable us to forget about what events the different Aludel implementations emit.

Otherwise we would have to manually check the Aludel implementations to ensure they emit the appropiate events, and we could potentially miss it.

Reply:
I was thinking in the subgraph because we used both InstanceAdded and AludelCreated in the subgraph mappings.

[InstanceAdded](https://github.com/alchemistcoin/crucible-subgraph/blob/aludel_factory/src/aludelFactory.ts#L8)
[AludelCreated](https://github.com/alchemistcoin/crucible-subgraph/blob/aludel_factory/src/aludelv1.5.ts#L29)

I agree with with your reasoning about emitting the event in the factory.

I don't really know about what event the frontend expects the contract to emit tbh, maybe @joehquak can throw some light here?